src/index.js

```diff
@@ -0,0 +1,3 @@
+import StockX from './apis/stockx';
+
+export { StockX };
```

Comment:
```suggestion
export default StockX;
```

We will need to go through and update the README to represent a default import, however.